integration/service/create_test.go

```diff
@@ -115,9 +116,33 @@ func TestCreateServiceMultipleTimes(t *testing.T) {
 	err = client.ServiceRemove(context.Background(), serviceID2)
 	assert.NilError(t, err)
 
+	// we can't just wait on no tasks for the service, counter-intuitively.
+	// Tasks may briefly exist but not show up, if they are are in the process
+	// of being deallocated. To avoid this case, we should retry network remove
+	// a few times, to give tasks time to be deallcoated
 	poll.WaitOn(t, swarm.NoTasksForService(ctx, client, serviceID2), swarm.ServicePoll)
 
-	err = client.NetworkRemove(context.Background(), overlayID)
+	for retry := 0; retry < 5; retry++ {
```

Comment:
I guess it's ok "for now", but wondering if:

- this should come handy in the `integration/internal/network` package
- is needed in the actual daemon (as this would likely not be something only occurring in tests)